test/LargeTests/test_large.py

```diff
@@ -0,0 +1,51 @@
+import sys
+from os import path
+sys.path.append(path.dirname(path.dirname(path.dirname(path.abspath(__file__)))))
+from ProjectLib import XMLtoProject
+from TimeLoopLib import TreeDynamicTimeStepping
+import unittest
+import glob
+import os
+
+filepathExampleSetups = path.join(path.dirname(path.abspath(__file__)),"Test_Setups_large/*.xml")
+xml = glob.glob(filepathExampleSetups)
+errors = []
+if xml:
+    for xmlfile in xml:
+        resultsFilepath = path.join(path.dirname(path.abspath(__file__)),"Test_Setups_large/testoutputs/*.*")
```

Comment:
Please change throughoutly (For all variables use "name_convention" instead of "camelCase". The camelCase is for functions only
```suggestion
        results_filepath = path.join(path.dirname(path.abspath(__file__)),"Test_Setups_large/testoutputs/*.*")
```